.github/workflows/aws_deploy.yml

```diff
@@ -164,6 +163,49 @@ jobs:
             NEXT_PUBLIC_GROWTHBOOK_CLIENT_KEY=${{ inputs.app-growthbook-client-key }}
             NEXT_PUBLIC_INTERCOM_APP_ID=${{ inputs.app-intercom-app-id }}
 
+      - name: Scan built image with Inspector
+        uses: aws-actions/vulnerability-scan-github-action-for-amazon-inspector@v1
+        id: inspector
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: ${{ inputs.ecr-repository }}
+          IMAGE_TAG: ${{ github.ref_name }}_${{ github.sha }}
+        with:
+          artifact_type: "container"
+          artifact_path: "${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ env.IMAGE_TAG }}"
+          critical_threshold: 5
+          high_threshold: 10
+
+      # Display Inspector results in the GitHub Actions terminal
+      - name: Display CycloneDX SBOM (JSON)
+        run: cat ${{ steps.inspector.outputs.artifact_sbom }}
+
+      - name: Display Inspector vulnerability scan results (JSON)
+        run: cat ${{ steps.inspector.outputs.inspector_scan_results }}
+
+      # Upload Inspector outputs as a .zip that can be downloaded
+      # from the GitHub actions job summary page.
+      - name: Upload Scan Results
+        id: upload-scan-results
+        uses: actions/upload-artifact@v4
+        with:
+        path: |
+          ${{ steps.inspector.outputs.inspector_scan_results }}
+          ${{ steps.inspector.outputs.inspector_scan_results_csv }}
+          ${{ steps.inspector.outputs.inspector_scan_results_markdown }}
+          ${{ steps.inspector.outputs.artifact_sbom }}
```

Comment:
question: where is this being uploaded and where can we see the content?

Reply:
github actions [summary](https://github.com/opengovsg/isomer/actions/runs/15869259433?pr=1476); alternatively, you can click onto the step itself and the artifact link will be there!

fun fact: this is also what we use to uploda e2e test artifacts